Makefile

```diff
@@ -1,16 +1,21 @@
 SHELL=/bin/bash
+GO?=go
+
+.PHONY: go gobuild gotest golint
+.PHONY: pf4 pf4build pf4test pf4lint
+.PHONY: storybook
 
 godep:
 	glide up
```

Comment:
this should be adapted too, or removed ; what's the process when we upgrade a dependency? We don't have to refresh anything explicitly anymore, right?

Reply:
Ops! Yes, I want to remove this :)

To upgrade a dependency:
* Run `go get dependency@version` (you can omit the `@version` part) within the project.
  * It will download the requested version and any sub-deps and update `go.mod` and `go.sum` as needed.
* Run `make go` to check that everything still works.
* Thereafter, you are supposed to run `go mod tidy` to reduce the number of injected subdependencies
  * But there is an issue with the `k8s.io/apimachinery` dependency. For some reason, when trying to tidy up dependencies, it grabs some `v0.18.*` version which breaks k-charted. So, I think we cannot run `go mod tidy` until that dependency is properly upgraded.
* Commit and you are done

Hmm... I'm not sure what you meant with "refresh" things.

Reply:
Ok, so `glide up` doesn't need to be replaced, it can just be removed if I understand correctly. That was what I meant by "refreshing" dependencies.